src/main/webapp/i18n/de/tutorialGroupImport.json

```diff
@@ -6,31 +6,31 @@
-                "requirements": "Jede Zeile muss immer einen Wert für den Übungsgruppen Titel enthalten. Wenn nicht nur Übungsgruppen erstellt werden sollen, sondern auch die Registrierungen von Studierenden, muss die Zeile einen Wert für die Immatrikulationsnummer oder für den Benutzernamen enthalten. Dies ist erforderlich, um den Studierenden zu identifizieren. Werte für den Nachnamen und Vornamen sind optional. Die erste Zeile muss die Spaltennamen angeben!"
+                "description": "Dieses Dialogfeld ermöglicht es, Tutoriumsgruppen und Studierendenregistrierungen aus einer CSV-Datei zu importieren. Wenn eine Tutoriumsgruppe noch nicht existiert, wird sie erstellt. Wenn Studierende in der CSV-Datei aufgeführt sind, werden sie von ihrer aktuellen Tutoriumsgruppe in die neue, in der CSV-Datei angegebene, Gruppe verschoben.",
+                "requirements": "Die oberste Zeile der CSV-Datei muss die Spaltennamen angeben. Jede Zeile muss den Titel der Tutoriumsgruppe enthalten. Um Studenten zu registrieren, muss jede Zeile zusätzlich entweder eine Matrikelnummer oder einen Anmeldenamen enthalten. Vor- und Nachname sind optional."
             },
             "supportedHeaderNames": {
                 "label": "Unterstützte Spaltennamen (Versionen mit Leerzeichen, Unterstrichen, Bindestrichen und Grossbuchstaben werden ebenfalls unterstützt): ",
-                "alwaysRequired": "Immer erforderlich (um die Übungsgruppe zu identifizieren): ",
-                "requiredIfStudentRegistration": "Eines davon ist erforderlich, wenn Studierende registriert werden sollen (um Studierende zu identifizieren): ",
-                "alwaysOptional": "Immer optional (nur für Anzeigzwecke): ",
-                "tutorialGroupTitle": "Für den Tiel der Übungsgruppe: {{ headers }}",
-                "registrationNumber": "Für die Immatrikulationsnummer: {{ headers }}",
-                "login": "Für den Benutznermane: {{ headers }}",
-                "firstName": "Für den Vornamen: {{ headers }}",
-                "lastName": "Für den Nachnamen: {{ headers }}"
+                "alwaysRequired": "Eins von den folgenden ist als Spaltennamen immer erforderlich um die Übungsgruppe zu identifizieren: ",
+                "requiredIfStudentRegistration": "Eins von den folgenden ist als Spaltennamen erforderlich, wenn Studierende identifiziert und registriert werden sollen: ",
+                "alwaysOptional": "Folgende Spaltennamen sind optional (nur für Anzeigzwecke): ",
+                "tutorialGroupTitle": "gruppe, titel, group, title -> Nach dem Parsen der CSV wird der Spaltenname als Übungsgruppe angezeigt",
+                "registrationNumber": "registrationnumber, matriculationnumber, matrikelnummer, number -> Nach dem Parsen der CSV wird der Spaltenname als Matrikelnummer angezeigt",
+                "login": "login, user, username, benutzer, benutzername -> Nach dem Parsen der CSV wird der Spaltenname als Anmeldename angezeigt",
```

Comment:
Is "benutzer" a fixed constant or could we stay with the English variant for better inclusiveness?

Reply:
👍 done